projects/aca-content/folder-rules/src/rule-details/conditions/rule-simple-condition.ui-component.ts

```diff
@@ -102,6 +127,11 @@ export class RuleSimpleConditionUiComponent implements ControlValueAccessor, OnD
   get isComparatorHidden(): boolean {
     return comparatorHiddenForConditionFieldType.includes(this.selectedField?.type);
   }
+
+  get fieldControl(): AbstractControl {
```

Comment:
could you please add 
```ts
<string>
```
to returned type? As it is control of string if you check definition of form: 
<img width="799" alt="Screenshot 2023-10-06 at 11 20 10" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/d29a6197-5996-4f63-b846-75114d145443">

Reply:
This getter has been removed completely. No need for this change